kolena/workflow/_datatypes.py

```diff
@@ -231,6 +231,10 @@ def deserialize_field(
                     items[key] = _try_deserialize_typed_dataobject(val)
         return cls(**items)
 
+    # integrate with pandas json deserialization
```

Comment:
I had trouble finding the documentation on this -- sharing for anyone else: https://pandas.pydata.org/docs/user_guide/io.html#fallback-behavior